### PR TITLE
[CBRD-23496] Invisible index manual

### PR DESCRIPTION
**Pull request by lucianlazar2211**
http://jira.cubrid.org/browse/CBRD-23496

Added information about the new invisible index and the `status` column.

### CODE REVIEW

File: en/sql/schema/index_stmt.rst [hunk 1]
```diff
@@ -20,7 +20,7 @@ For how to use indexes on the **SELECT** statement like Using SQL Hint, Descendi
         <index_col_desc> ::=
             { ( column_name [ASC | DESC] [ {, column_name [ASC | DESC]} ...] ) [ WHERE <filter_predicate> ] | 
             (function_name (argument_list) ) } 
-                [WITH ONLINE [PARALLEL parallel_count]]
+                [WITH ONLINE [PARALLEL parallel_count] | INVISIBLE]
```
Comment (razvanradulescu) [inline en/sql/schema/index_stmt.rst]:
```suggestion
                { [WITH ONLINE [PARALLEL parallel_count]] | INVISIBLE | VISIBLE }
```

File: en/sql/catalog.rst [hunk 2]
```diff
@@ -473,6 +473,8 @@ Represents index information. An index for class_of is created.
 +--------------------+---------------------------+------------------------------------------------+
 | comment            | VARCHAR (1024)            | Comment to describe the index                  |
 +--------------------+---------------------------+------------------------------------------------+
+| status             | INTEGER                   | Index status                  |
```
Comment (ribram) [inline en/sql/catalog.rst]: fix row's last column

Comment (razvanradulescu) [inline en/sql/catalog.rst]: please make sure the this row in table if properly formatted (in github view it appears it requires more spaces)

File: en/sql/query/show.rst [hunk 3]
```diff
@@ -215,6 +215,7 @@ Null                                VARCHAR         YES if a column can include
 Index_type                          VARCHAR         Index to be used (currently, only the BTREE is supported.)
 Func                                VARCHAR         A function which is used in a function-based index
 Comment                             VARCHAR         Comment to describe the index
+Status                              VARCHAR         Shows the status of an index (NORMAL_INDEX|INVISIBLE|ONLINE_INDEX_IN_PROGRESS)
```
Comment (ribram) [inline en/sql/query/show.rst]: doesn't match with what query outputs:

Visible: 'YES'

File: en/sql/schema/index_stmt.rst [hunk 4]
```diff
@@ -229,7 +231,7 @@ Online unique index while other transactions inserts violates uniqueness
 ALTER INDEX
 ===========
 
-The **ALTER INDEX** statement rebuilds an index or adds/changes the comment of an index. Rebuilding an index is a job which drops and recreates an index.
+The **ALTER INDEX** statement rebuilds an index or adds/changes the comment/status of an index. Rebuilding an index is a job which drops and recreates an index.
```
Comment (ribram) [inline en/sql/schema/index_stmt.rst]: Please rephrase more clearly. Something along the lines:
```suggestion
The **ALTER INDEX** statement changes the properties of an index. Index is rebuilt unless only comment or status is changed. Rebuilding an index is a job which drops and recreates an index.
```

File: en/sql/schema/index_stmt.rst [hunk 5]
```diff
@@ -273,10 +275,18 @@ The below is a syntax to only add or change a comment without rebuilding an inde
 
 The following is a syntax of renaming an index.
 
-:: 
+.. code-block:: sql
 
     ALTER INDEX old_index_name ON table_name RENAME TO new_index_name [COMMENT 'index_comment_string'] ;
 
+The following is a syntax to change the status of an index to **INVISIBLE**/**VISIBLE**. When an index is set as **INVISIBLE**, queries will be executed as the index does not exist. In this way, you can check if the index actually provides an increase in performance by check if the query execution time drops if the index is not used, without actually dropping the index.
```
Comment (ribram) [inline en/sql/schema/index_stmt.rst]:
```suggestion
The following is a syntax to change the status of an index to **INVISIBLE**/**VISIBLE**. When an index is set as **INVISIBLE**, queries will be executed as the index does not exist. In this way, the performance of the index may be tested and the impact of its removal evaluated without actually dropping the index.
```

**Pull request merged**